pkg/cmd/codespace/rebuild.go

```diff
@@ -0,0 +1,56 @@
+package codespace
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cli/cli/v2/internal/codespaces/api"
+	"github.com/spf13/cobra"
+)
+
+func newRebuildCmd(app *App) *cobra.Command {
+	var codespace string
+
+	rebuildCmd := &cobra.Command{
+		Use:   "rebuild",
+		Short: "Rebuild a codespace",
+		Args:  cobra.NoArgs,
```

Comment:
Would be great to support -c at least

Reply:
`cobra.NoArgs` means no positional args (`gh cs rebuild positionalArg -flag flagArg`). This supports `-c`, see a few lines down before `return rebuildCmd`

Reply:
🙈 oops makes sense